### ISSUE TITLE
Navigating notification a11y view with alt+]/[ should announce where you are in the list

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#188765]
Testing #188528

I'm expecting something like "notification x of y", instead it just announces the next focused notification, even if it's the same one:

![image](https://github.com/microsoft/vscode/assets/2193314/929ae59f-a833-44c8-952e-7b69dc528c21)

**Comment by meganrogge** [comment]
We should also do this for chat responses https://github.com/microsoft/vscode/issues/188831#issuecomment-1653763626

**Comment by meganrogge** [comment]
@rperez030 would be amazing if you could verify this 🙏🏼 

Note that we will improve this in August https://github.com/microsoft/vscode/issues/189105

**Comment by rperez030** [comment]
Verified in main. It works as expected.
